IdentityCore/src/cache/crypto/MSIDAssymetricKeyKeychainGenerator.m

```diff
@@ -299,6 +299,8 @@ - (void)logAndFillError:(NSString *)errorTitle status:(OSStatus)status error:(NS
     {
         *error = MSIDCreateError(MSIDErrorDomain, MSIDErrorInternal, description, nil, nil, nil, nil, nil, NO);
     }
+    
+    return YES;
```

Comment:
Does it always return ```YES```?

Reply:
yes, the analyzer complained about Apple code style: the method that fills error should return a boolean that indicates whether error was filled or not.